### PR TITLE
feat(odyssey-react): add Link component

### PR DESCRIPTION
**Pull request by arnoldsandoval-okta**

### CODE REVIEW

**Review by edburyenegren-okta — APPROVED** [review]
LGTM

File: packages/odyssey-react/src/components/Link/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  return (
+    <a 
+      {...omitProps}
+      target={target}
+      rel={rel}
+      href={href}
+      className="ods-link"
+    >
+      {children}
+    </a>
```
Comment (jeffbelser-okta) [inline packages/odyssey-react/src/components/Link/index.tsx:65]: The visual indication we place here in a pseudo-element for external links isn't accessible to assistive tech so we should consider a text based alternative going forward

**Review by jeffbelser-okta — APPROVED** [review]
strong work 💪

File: packages/odyssey-react/src/components/Link/index.tsx (hunk 1 above)
Comment (arnoldsandoval) [reply packages/odyssey-react/src/components/Link/index.tsx:65]: The external link visual indicator doesn't need to be announced accessible user agents as its display is dependent on the presence of the `target` and/or `rel` attributes -- am I missing something?

**Pull request merged**